venues/ICML.cc/2018/Workshop/NAMPI/python/setup.ipynb

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
please don't push a notebook, I can not follow the code here

Reply:
if it is 'one-time-use' then we don't need to commit the code then. 

I can not do code review of a python notebook.